docs/onboarding/8_16/operator/troubleshoot-auto-instrumentation.md

```diff
@@ -0,0 +1,85 @@
+# Troubleshooting auto-instrumentation
+
+1. Check the operator is running, eg
+```
+$ kubectl get pods -n opentelemetry-operator-system
+NAME                                                              READY   STATUS             RESTARTS      AGE
+opentelemetry-kube-stack-opentelemetry-operator-7b8684cfbdbv4hj   2/2     Running            0             58s
+...
+```
+
+2. Check the instrumentation has been installed, eg
+```
+$ kubectl describe Instrumentation -n opentelemetry-operator-system
+Name:         elastic-instrumentation
+Namespace:    opentelemetry-operator-system
+  ...
+Kind:         Instrumentation
+Metadata:
+  ...
+Spec:
+  Dotnet:
+    Image:  docker.elastic.co/observability/elastic-otel-dotnet:edge
+  Go:
+    Image:  ghcr.io/open-telemetry/opentelemetry-go-instrumentation/autoinstrumentation-go:v0.14.0-alpha
+  Java:
+    Image:  docker.elastic.co/observability/elastic-otel-javaagent:1.0.0
+  Nodejs:
+    Image:  docker.elastic.co/observability/elastic-otel-node:edge
+  Python:
+    Image:  docker.elastic.co/observability/elastic-otel-python:edge
+  ...
+```
```

Comment:
I would include an step to check the `Instrumentation` endpoint and ensure it's reachable (and resolvable) by the Pod.
If for example the endpoint is incorrect or not resolvable, the Pod will fail to send traces, and I think it will be a common use case.